### ISSUE TITLE
Delete action button not working?

### ISSUE THREAD

**Issue by bpirtle-nova** [activeadmin-plugins/active_admin_scoped_collection_actions#29]
I added the basic batch delete action and the sidebar appears, yet when I click the button nothing happens. It doesn't submit the form as I would expect and perform the deletions. Here is the HTML of the button:

<button class="scoped_collection_action_button" data="{&quot;auth_token&quot;:&quot;jA1MyIjnRTUnUVkzvXIWQY6zHjDqRcUemIS3I8dc7a+MKAqPUOdYaFurC4yNjnYBz6uzWyzXuLzLqS+JG5LTSw==&quot;,&quot;batch_action&quot;:&quot;scoped_collection_destroy&quot;,&quot;confirm&quot;:&quot;Are you sure?&quot;}">Delete batch</button>

Any idea why the button is non-functional?

**Comment by bpirtle-nova** [comment]
This is my AA version and associated gems from gemfile:

gem 'activeadmin', '~> 1.0.0.pre2'

gem 'active_admin_datetimepicker'

gem 'active_admin_scoped_collection_actions'

**Comment by workgena** [comment]
@bpirtle-nova Tested with demo application and it works. Make sure JS and CSS [included](https://github.com/activeadmin-plugins/active_admin_scoped_collection_actions#install) properly

app/assets/javascript/active_admin.js.coffee
```#= require active_admin_scoped_collection_actions```

**Comment by bpirtle-nova** [comment]
So all that was correct, but after some extensive verification I found the following caused the issue:

I am also using this gem: https://github.com/shekibobo/xdan-datetimepicker-rails but it has a bug in the current version that can be seen here: https://github.com/xdan/datetimepicker/issues/392

As a result, all of the JS for active admin is compiled into a single JS file for Heroku (Rails asset pipeline thing) and this file was crashing due to that bug. Since in that single JS file the datetimepicker code was trying to load first, the undhandled exception caused this JS to not load at all!

I found that version 2.4.5 of this lib doesn't crash and now everything works fine. Quite . the nasty surprise :(
